Firefox crashes very early in start if using nVidia hardware and
driver 470.x or higher.  The problem appears that Xorg and/or Mesa doesn't
have working EGL support on OI, even though the nVidia driver claims support.
Make sure EGL is not used for now.  Revisit patch when xorg and mesa are
updated on OI.

--- firefox-114.0/toolkit/xre/glxtest/glxtest.cpp.old	2023-01-25 16:57:32.381884245 +0000
+++ firefox-114.0/toolkit/xre/glxtest/glxtest.cpp	2023-01-25 17:01:49.335385472 +0000
@@ -478,6 +478,9 @@
 
 static bool get_egl_gl_status(EGLDisplay dpy,
                               PFNEGLGETPROCADDRESS eglGetProcAddress) {
+
+  return false;
+
   typedef EGLBoolean (*PFNEGLCHOOSECONFIGPROC)(
       EGLDisplay dpy, EGLint const* attrib_list, EGLConfig* configs,
       EGLint config_size, EGLint* num_config);
@@ -637,6 +640,9 @@
 }
 
 static bool get_egl_status(EGLNativeDisplayType native_dpy) {
+
+  return false;
+
   log("GLX_TEST: get_egl_status start\n");
 
   EGLDisplay dpy = nullptr;
@@ -917,6 +923,9 @@
 }
 
 bool x11_egltest() {
+
+  return false;
+
   log("GLX_TEST: x11_egltest start\n");
 
   Display* dpy = XOpenDisplay(nullptr);
@@ -1223,9 +1232,7 @@
 #ifdef MOZ_X11
   if (!aWayland) {
     // TODO: --display command line argument is not properly handled
-    if (!x11_egltest()) {
       glxtest();
-    }
   }
 #endif
   // Finally write buffered data to the pipe.
